src/patternfly/components/DataList/data-list.scss

```diff
@@ -238,6 +249,57 @@
     box-shadow: var(--pf-c-data-list__item--m-selected--BoxShadow);
   }
 
+  // stylelint-disable
+  &.pf-m-draggable {
+    position: relative;
+
+    .pf-c-data-list__item-draggable-button {
+      padding-right: var(--pf-c-data-list__item--m-draggable__draggable-button--PaddingRight);
+      padding-left: var(--pf-c-data-list__item--m-draggable__draggable-button--PaddingLeft);
+      margin-top: var(--pf-c-data-list__item--m-draggable__draggable-button--MarginTop);
+      margin-left: var(--pf-c-data-list__item--m-draggable__draggable-button--MarginLeft);
+      border: 0;
+
+      &:hover {
+        cursor: grab;
+
+        --pf-c-data-list__item--m-draggable__draggable-button__draggable-icon--Color: var(--pf-c-data-list__item--m-draggable__draggable-button--hover__draggable-icon--Color);
+      }
+
+      &:active {
+        cursor: grabbing;
+      }
+
+      &.pf-m-disabled {
+        pointer-events: none;
+        cursor: none;
+
+        --pf-c-data-list__item--m-draggable__draggable-button__draggable-icon--Color: var(--pf-c-data-list__item--m-draggable__draggable-button--m-disabled__draggable-icon--Color);
+      }
+
+      .pf-c-data-list__item-draggable-icon {
+        color: var(--pf-c-data-list__item--m-draggable__draggable-button__draggable-icon--Color);
+      }
+    }
+
+    &.pf-m-ghost-row {
+      .pf-c-data-list__item-row {
+        &::after {
```

Comment:
This works, assuming we'll close any expanded content sections when you click on the draggable button. Though it could also apply to `.pf-c-data-list__item::after`, and it would overlay anything that's in the `__item`. 

An alternative to this could also be just to change the `opacity` on the item.

Or using a white as a background color and `opacity`, or just `rgba(255,255,255,0.8)` as a background color. 

Is there a benefit to using `mix-blend-mode` instead?

Reply:
im not sure of the benefit to using `mix-blend-mode` instead, so I think your idea is a nice simple, straight-forward approach!